jdaviz/configs/imviz/helper.py

```diff
@@ -573,5 +569,10 @@ def link_image_data(app, link_type='pixels', wcs_fallback_scheme='pixels', wcs_u
             'Images successfully relinked', color='success', timeout=8000, sender=app))
 
     if link_plugin is not None:
+        # Only broadcast after success.
+        app.hub.broadcast(LinkUpdatedMessage(link_type,
+                                             wcs_fallback_scheme == 'pixels',
+                                             wcs_use_affine,
+                                             sender=app))
```

Comment:
this move should be ok since the spinner state is updated independently of the event, the only difference will be that when calling linking from the API the spinner will now start, the linking will happen, THEN the radio buttons will update and the spinner will clear (whereas before the radio buttons would switch before the linking actually started).  I can see how for other uses of checking the linking (like in this PR) it is more beneficial to receive the message after the linking has been completed.